sdks/python/apache_beam/io/gcp/spanner_wrapper.py

```diff
@@ -0,0 +1,73 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the 'License'); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import logging
+import uuid
+import apache_beam as beam
+
+try:
+  from google.cloud import spanner
+
+except ImportError:
+  spanner = None
+
+_LOGGER = logging.getLogger(__name__)
+
+class SpannerWrapper(object):
+  TEST_DATABASE = None
+  TEMP_DATABASE_PREFIX = 'temp-'
+  _SPANNER_CLIENT = None
+  _SPANNER_INSTANCE = None
```

Comment:
I have no idea why they wrote the BQ wrapper this way, but these don't need to be class variables as they are state variables (except `TEMP_DATABASE_PREFIX`, I suppose) I would remove at least the state variables and move to constructor as instance variables.
```suggestion
  TEMP_DATABASE_PREFIX = 'temp-'
```